lib/slack/lookups.ex

```diff
@@ -47,13 +53,20 @@ defmodule Slack.Lookups do
   @doc ~S"""
   Turns a Slack user ID (`"U…"`), direct message ID (`"D…"`), or bot ID (`"B…"`)
   into a string in the format "@USER_NAME".
+
+  NOTE: Referencing `"@USER_NAME"` is deprecated, and should not be used.
```

Comment:
I like that we added this to the documentation. How do you feel about logging a deprecation warning pointing to the new methods?

Reply:
I added logging statement with essentially the same deprecation warning as the docs.